docConfig.json

```diff
@@ -49,6 +49,7 @@
             { "name": "{PackageVerChanges-24-1-JUN}", "value": "18.0.0 (June 2024)"  },
             { "name": "{PackageVerChanges-24-1-SEP}", "value": "18.1.0 (September 2024)"  },
             { "name": "{PackageVerChanges-24-2-DEC}", "value": "18.2.0 (December 2024)" },
+            { "name": "{PackageVerChanges-24-2-JAN}", "value": "19.0.0 (January 2025)" },            
```

Comment:
add "{PackageVerChanges-24-2-JAN}" variable to other platforms in this config

Reply:
Please clarify, there are no other new versions. please merge

Reply:
either build flag usage of that new variable for Angular OR you must add "{PackageVerChanges-24-2-JAN}" variable to other platforms with value matching previously release version in this config because this topic is shared between all platforms and currently this variable will not be resolved to any value for Blazor, React and WC platforms